BREAKING_CHANGES_LOG.md

```diff
@@ -3,7 +3,22 @@ Before 1.0, the stack do NOT follow semver version in releases.
 This document aims to ease the WIP migration from a version to another by providing intels about what to do to migrate.
 
 ## v0.153.0
-* CMF: HeaderBar
+* Component: SubHeaderBar
+* PR: https://github.com/Talend/ui/pull/1041
+* Changes: props components has change
+
+| name | change |
+|---|---|
+| props.components | replaced by the new Inject API |
+| props.left | added with the same API has the ActionBar |
```

Comment:
those added element are hardly a breaking change did you remove another props for it ?

Reply:
yes they replace in fact the components which was the only entry point.